<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>351</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
	image: url(:/source/clock.svg);
}</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border-image:url(:/source/button.svg); }
QPushButton:hover { border-image:url(:/source/button_hover.svg); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>124</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::RightToLeft</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>240</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border-image:url(:/source/button2.png); }
QPushButton:hover { border-image:url(:/source/button2_hover.svg); }
QPushButton:pressed { border-image:url(:/source/button2_hover.svg); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_2{
image: url(:/source/seting_screen_2.svg);
}</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>35</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border-image:url(:/source/close.svg); }
QPushButton:hover { border-image:url(:/source/close_gray.svg); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>35</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border-image:url(:/source/30.svg); }
QPushButton:hover { border-image:url(:/source/30_gray.svg); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>35</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border-image:url(:/source/60.svg); }
QPushButton:hover { border-image:url(:/source/60_gray.svg); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
